set.mm

```diff
@@ -21358,8 +21374,8 @@ proposition with a distinct variable (closed form of ~ nfsb4 ).
   ${
     sbie.1 $e |- F/ x ps $.
     sbie.2 $e |- ( x = y -> ( ph <-> ps ) ) $.
-    $( Conversion of implicit substitution to explicit substitution.  For a
-       version requiring disjoint variables, but fewer axioms, see ~ sbiev .
+    $( Conversion of implicit substitution to explicit substitution.  For
+       versions requiring disjoint variables, but fewer axioms, see ~ sbiev .
```

Comment:
"..., see ~ sbiev and ~ sbievw ."

Reply:
I am going to fix this in an extra pull request, so development is not blocked by this minor issue.